### PR TITLE
[zh-cn]Sync PodDisruptionsCondition to PodDisruptionConditions

### PR DESCRIPTION
**Pull request by yanrongshi**

### CODE REVIEW

File: content/zh-cn/docs/concepts/workloads/pods/disruptions.md [hunk 1]
```diff
@@ -586,11 +585,15 @@ the nodes in your cluster, such as a node or system software upgrade, here are s
...
 * Follow steps to protect your application by [configuring a Pod Disruption Budget](/docs/tasks/run-application/configure-pdb/).
+
 * Learn more about [draining nodes](/docs/tasks/administer-cluster/safely-drain-node/)
+
 * Learn about [updating a deployment](/docs/concepts/workloads/controllers/deployment/#updating-a-deployment)
   including steps to maintain its availability during the rollout.
 -->
 * 参考[配置 Pod 干扰预算](/zh-cn/docs/tasks/run-application/configure-pdb/)中的方法来保护你的应用。
+
 * 进一步了解[排空节点](/zh-cn/docs/tasks/administer-cluster/safely-drain-node/)的信息。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/pods/disruptions.md:596]:
```suggestion
* 进一步了解[清空节点](/zh-cn/docs/tasks/administer-cluster/safely-drain-node/)的信息。
```
Following the term after you click is a good idea.

File: content/zh-cn/docs/tasks/job/pod-failure-policy.md [hunk 2]
```diff
@@ -237,7 +237,7 @@ The cluster automatically cleans up the Pods.
 You could rely solely on the
 [Pod backoff failure policy](/docs/concepts/workloads/controllers/job#pod-backoff-failure-policy),
 by specifying the Job's `.spec.backoffLimit` field. However, in many situations
-it is problematic to find a balance between setting the a low value for `.spec.backoffLimit`
+iit is problematic to find a balance between setting a low value for `.spec.backoffLimit`
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/job/pod-failure-policy.md:240]: ?

**Review by windsonsea — CHANGES_REQUESTED** [review]

**Comment by Sea-n** [comment]
The `content/zh-cn/docs/tasks/job/pod-failure-policy.md` file have some additional changes, please resync it again. Thanks!

**Comment by yanrongshi** [comment]
> The `content/zh-cn/docs/tasks/job/pod-failure-policy.md` file have some additional changes, please resync it again. Thanks!

/ok，thanks

**Comment by tengqm** [comment]
Where are we now?

**Comment by windsonsea** [comment]
Ping @yanrongshi to check `content/zh-cn/docs/tasks/job/pod-failure-policy.md` again.

**Pull request closed without merging**